internal/k8s/informers.go

```diff
@@ -48,8 +48,9 @@ func IngressV1Resources() []schema.GroupVersionResource {
 	}
 }
 
-// +kubebuilder:rbac:groups="networking.x-k8s.io",resources=gatewayclasses;gateways;httproutes;backendpolicies;tlsroutes;tcproutes;udproutes,verbs=get;list;watch
-// +kubebuilder:rbac:groups="networking.x-k8s.io",resources=gatewayclasses/status;httproutes/status;backendpolicies/status;tlsroutes/status;tcproutes/status;udproutes/status,verbs=update
+// +kubebuilder:rbac:groups="networking.x-k8s.io",resources=gatewayclasses;gateways,verbs=get;list;watch;update
```

Comment:
is `update` needed on these objects? or just on their status (which is covered in L53)?

Reply:
Ahh not specifically, those were for the finalizers bit. I'll clean up for now.